### DISCUSSION TITLE
Prettier not respecting cli-overrides

### DISCUSSION THREAD

**Discussion by ghost** [zed-industries/zed#14214] [Feature Requests]
As the title says, are there any reasons why this is not working? I have tried manually formatting it through the CLI with the following overrides, and it works as expected. However, when using editor:format in Zed, it does not respect these settings. Am I doing something wrong?

```json
  "formatter": {
    "external": {
      "command": "prettier",
      "arguments": [
        "--stdin-filepath",
        "{buffer_path}",
        "--print-width 160",
        "--tab-width 4"
      ]
    }
  }
```

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
